samples/CosmosDB/audit-cosmosdb-throughput/azurepolicy.json

```diff
@@ -0,0 +1,138 @@
+{
+  "properties": {
+    "displayName": "Audit Cosmos DB Throughput Exceeding Max",
+    "policyType": "Custom",
+    "description": "This policy audits when Cosmos DB shared or dedicated throughput exceeds max",
+    "mode": "All",
+    "metadata": {
+      "category": "Cosmos DB"
+    },
+    "parameters": {
+      "throughputMax": {
+        "type": "integer",
+        "defaultValue": 400,
+        "metadata": {
+          "displayName": "Max RUs",
+          "description": "Max RUs"
```

Comment:
>"description": "Max RUs" [](start = 10, length = 24)

Descriptions should either provide additional info or be excluded.  A description that is the same as the display name isn't necessary.